assets/src/blocks/Spreadsheet/SpreadsheetFrontend.js

```diff
@@ -187,9 +187,15 @@ export class SpreadsheetFrontend extends Component {
                           )
                         }
                         onClick={ () => { this.onHeaderClick( index ) } }
-                        key={ index } title={ __('Sort by', 'planet4-blocks') }>
-                        { cell }
-                        <ArrowIcon />
+                        key={ index }
+                        scope='col'
+                        title={ __(cell) }>
```

Comment:
The `__` function is only needed for localizing English strings in our source code, which doesn't need to happen for the spreadsheet data since it's already expected to be the right language.

Reply:
Thanks. Fixed.